includes/class-sensei-preview-user.php

```diff
@@ -244,6 +254,28 @@ private function can_switch_to_preview_user( $course_id ) {
 		return Sensei_Course::can_current_user_edit_course( $course_id );
 	}
 
+	/**
+	 * Prevent emails related to the preview user from being dispatched via wp_mail.
+	 *
+	 * @access private
+	 * @since  $$next-version$$
+	 *
+	 * @param bool|null $return Null if we should send the email, a boolean if not.
+	 * @param array     $atts   Email attributes.
+	 * @return bool|null Null if we should send the email, a boolean if not.
+	 */
+	public function skip_wp_mail( $return, $atts ) {
+		if ( $this->is_preview_user_active() ) {
+			// If this e-mail is being dispatched while the current user is a previwe user, just... don't send it.
+			return false;
+		}
+		if ( Sensei_Temporary_User::should_block_email( $atts, self::EMAIL_DOMAIN ) ) {
+			// If this e-mail is being dispatched to a preview user, don't send it.
+			return false;
+		}
```

Comment:
Similar to the Guest Users situation, in this block, we're returning `false` so `wp_mail` returns `false` too. This is considered an "error". It might not be the most friendly way of doing this (for any third party developer debugging this), but I believe it is better than returning `true` and faking a possible success..